core/spec/models/spree/concerns/display_money_spec.rb

```diff
@@ -16,7 +17,21 @@ def total
 
       context 'currency is not defined' do
         it 'generates a display_method that builds a Spree::Money without options' do
-          expect(test_class.new.display_total).to eq Spree::Money.new(10.0)
+          expect(test_instance.display_total).to eq Spree::Money.new(10.0)
+        end
+        context 'wrapped method accepts `:currency` keyword argument' do
+          let(:test_class) do
+            Class.new do
+              extend DisplayMoney
+              def total(currency:)
+                10.0
+              end
+            end
+          end
+          let(:currency) { 'GBP' }
```

Comment:
Please add new line.

```suggestion
          let(:currency) { 'GBP' }
          
```